README.md

```diff
@@ -230,8 +230,8 @@ App behavior parameters
   - Default: `false`
 - `enableValidator`: Enables or disables the built-in HTML validator in dev mode.
   - Default: `false`
-- `htmlValidator`: Params to send to [html-validator](https://github.com/zrrrzzt/html-validator#usage) (if `enableValidator` is set to true).
-  - Default:  `{validator: 'http://html5.validator.nu', format: 'text'}`
+- `htmlValidator`: Params to send to [html-validator](https://github.com/zrrrzzt/html-validator#usage) (if `enableValidator` is set to true). When `suppressWarnings` is set to true validation warnings will be hidden.
```

Comment:
"When `suppressWarnings` is set to true validation warnings will be hidden and only errors will be shown."